### PR TITLE
Tbrands 28 carousel block - adding util function for extracting stories

### PR DESCRIPTION
**Pull request by brentmiller1973**
## Ticket

- Part of the PR process for [[TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TBRANDS-28)

## Description

Request in https://github.com/WPMedia/arc-themes-blocks/pull/1317 to add extractImageFromStory into the util folder of this repository.  Also reduced the test content sources and upped the branch metrics to comply with threshold.  It was renamed to getImageFromANS

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ import Stack from "./src/components/stack";
 import Video from "./src/components/video";
 import formatCredits from "./src/utils/format-credits";
 import formatPowaVideoEmbed from "./src/utils/format-powa-video-embed";
+import getImageFromANS from "./src/utils/get-image-from-ans/getImageFromANS";
```
Comment (matthewroach) [inline index.js]: Can we rename the file `getImageFromANS.js` to `index.js` and then we can do an image like the others 

```
import getImageFromANS from "./src/utils/get-image-from-ans";
```

**Review by matthewroach — CHANGES_REQUESTED** [review]
One small change to rename the files to allow for directory import vs direct file importing

File: index.js (hunk 1 above)
Comment (nschubach) [reply index.js]: /agree

Comment (brentmiller1973) [reply index.js]: @matthewroach , @nschubach Done!

**Review by nschubach — APPROVED** [review]

**Review by matthewroach — APPROVED** [review]

**Pull request merged**